src/main/java/com/github/manolo8/darkbot/core/entities/BattleStation.java

```diff
@@ -16,10 +16,11 @@
 public class BattleStation
         extends Entity
         implements Obstacle, eu.darkbot.api.game.entities.BattleStation {
+    private static final int AVOID_RADIUS = 700;
```

Comment:
are we sure that 700 is enough not to be hit? i think the original 1200 around center is more than the 700 around the module.

Also one big consideration: we need to be certain that the center does not become un-avoided, or else, we may create invalid paths that get the bot stuck (ie: the bot trying to navigate from outside the battle station to inside the middle of the station).
Are we certain that 700 radius overlaps enough?

Ontop of that, i'm pretty certain that at some point in the past this was already a problem, and i believe we switched to a unique region to reduce issues, can we double-check git history and confirm what happened in the past won't be an issue now?